### PR TITLE
feat: group log output

### PR DESCRIPTION
**Pull request by villelahdenvuo**
I added grouping to the log output to make it easier to go through the output.

I have my editor set to run prettier on save, so I hope you don't mind the formatting changes, I added a basic editorconfig file to try to keep the changes to a minimum.

Let me know if you want me to revert all the style changes or put them in a separate commit.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -42,8 +43,15 @@ const stream = __nccwpck_require__(2781);
 const ORIGIN = 'licensed-ci-origin';
 
 async function configureGit() {
-  const token = core.getInput('github_token', { required: true });
-  await exec.exec('git', ['remote', 'add', ORIGIN, `https://x-access-token:${token}@github.com/${process.env.GITHUB_REPOSITORY}`]);
+  await core.group('Configuring git', async () => {
```
Comment (jonabc) [inline dist/index.js:46]: oh this is really nice,  TIL!

**Review by jonabc — COMMENTED** [review]
> I have my editor set to run prettier on save, so I hope you don't mind the formatting changes, I added a basic editorconfig file to try to keep the changes to a minimum.

> Let me know if you want me to revert all the style changes or put them in a separate commit.

Style changes 😅 .  Can you put any style changes as a separate PR so I can take a look separately?  This repo isn't set up for prettier and I'm not personally a fan of much of its style enforcements, but always happy to reconsider if it would lower the barrier to entry for contributions 🚀 

The non-style changes look great and I wouldn't mind getting those merged in separately.

**Comment by jonabc** [comment]
Sorry for the delay, I missed the notification that you opened this PR 🙏

**Comment by villelahdenvuo** [comment]
No problem, I opened a new PR with (mostly) only the functional changes at #124

**Comment by villelahdenvuo** [comment]
> I'm not personally a fan of much of its style enforcements

I used to also hate some of the style choice of Prettier and wished they were more configurable, but after a while I got used to it and it's really nice to not have to worry about white space, semicolons, etc. as it's just handled automatically and there is no need to argue about style issues in PRs. 😄 

I can make a separate PR for a simple prettier config if you want, but if you don't like the idea it's okay.

**Comment by jonabc** [comment]
> after a while I got used to it and it's really nice to not have to worry about white space, semicolons, etc. as it's just handled automatically and there is no need to argue about style issues in PRs. 😄

😄  I don't bother with personal styling preferences on shared projects, but I get a bit pickier on projects I maintain.  If this project picks up more OSS contributions though I'd probably drop my personal preferences and/or set up the repo with more descriptive linting/formatting rules or tools.

**Pull request closed without merging**